components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1066,7 +1068,7 @@ func (p *pg) DeleteRule(ctx context.Context, id string) error {
 		res, err := tx.ExecContext(ctx,
```

Comment:
ℹ️ Thanks to the new helper function `projects_match_for_rule(INTEGER, TEXT[])`, the changes to `DeleteRule` at only drive-by nitpicks.